arch/arm/soc/nordic_nrf/nrf52/dts.fixup

```diff
@@ -23,10 +23,10 @@
 
 #define FLASH_DEV_NAME			NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL
 
-#define CONFIG_GPIO_NRF5_P0_DEV_NAME	NORDIC_NRF5_GPIO_50000000_LABEL
-#define CONFIG_GPIO_NRF5_P1_DEV_NAME	NORDIC_NRF5_GPIO_50000300_LABEL
-#define CONFIG_GPIOTE_NRF5_IRQ_PRI	NORDIC_NRF5_GPIOTE_40006000_IRQ_0_PRIORITY
-#define CONFIG_GPIOTE_NRF5_IRQ		NORDIC_NRF5_GPIOTE_40006000_IRQ_0
+#define CONFIG_GPIO_P0_DEV_NAME		NORDIC_NRF_GPIO_50000000_LABEL
+#define CONFIG_GPIO_P1_DEV_NAME		NORDIC_NRF_GPIO_50000300_LABEL
```

Comment:
nRF52832 does not have P1, AFAICS. Would it make sense to skip this define for that SOC, and leave it for 52840 only?

Reply:
It would require additional fixup file, and in current way it doesn't affect nRF52832 so I would keep it that way, especially that we are gonna get rid of it as soon as code generation gets merged.

Reply:
I understand that it does not affect nRF53832, I am wondering, now, why it would need a separate fixup? Could we just add #ifdef inside the fixup for nRF53840?

Reply:
Ok, I've added required if